applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationRawClient.java

```diff
@@ -0,0 +1,552 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.applicationconfig;
+
+import com.azure.applicationconfig.credentials.ConfigurationClientCredentials;
+import com.azure.applicationconfig.models.ConfigurationSetting;
+import com.azure.applicationconfig.models.SettingFields;
+import com.azure.applicationconfig.models.SettingSelector;
+import com.azure.core.ServiceClient;
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.exception.ResourceModifiedException;
+import com.azure.core.exception.ResourceNotFoundException;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.rest.PagedResponse;
+import com.azure.core.http.rest.Response;
+import com.azure.core.implementation.RestProxy;
+import com.azure.core.implementation.logging.ServiceLogger;
+import com.azure.core.implementation.util.ImplUtils;
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.lang.reflect.Constructor;
+import java.net.URL;
+import java.util.Objects;
+
+/**
+ * This class provides a client that contains all the operations for {@link ConfigurationSetting ConfigurationSettings}
+ * in Azure App Configuration Store. Operations allowed by the client are adding, retrieving, updating, and deleting
+ * ConfigurationSettings, and listing settings or revision of a setting based on a {@link SettingSelector filter}.
+ *
+ * <p><strong>Instantiating an Asynchronous Configuration Client</strong></p>
+ *
+ * <pre>
+ * ConfigurationAsyncClient client = ConfigurationAsyncClient.builder()
+ *     .credentials(new ConfigurationClientCredentials(connectionString))
+ *     .build();
+ * </pre>
```

Comment:
We will need to revisit all the reference docs if this change is made.

Reply:
Definitely will need something to make maintaining the examples easier.